recipes/mono.recipe

```diff
@@ -12,7 +13,11 @@ class Recipe(recipe.Recipe):
                         ' --disable-dtrace ' \
                         ' --with-mcs-docs=no '
     config_sh = 'sh autogen.sh'
-    files_bins = ['mono-sgen', 'mono', 'xbuild']
+    platform_files_bins = {
+        Platform.DARWIN: ['mono-sgen', 'mono', 'xbuild'],
```

Comment:
if xbuild is the only exception, use platform_files_bins only for this one and keep the others in files_bins